document/CoreFormat.mdk

```diff
@@ -355,12 +359,52 @@ An `issue` object describes a single issue detected by a static analysis tool.
 ### `ruleId` property { #issue-ruleId }
 
 An `issue` object shall contain a property named `ruleId` whose value is a string containing
-a stable, opaque identifier for the rule that was violated.
+the stable, opaque identifier for the rule that was violated.
 
 ~ IsoExample
 `"CA2101"`
 ~
 
+### `kind` property { #issue-kind }
+
+An `issue` object may contain a property named `kind` whose value is one of a fixed set of strings
+that specify what kind of issue this is.
+
+If present, the `kind` property shall have one of the following values, with the specified meanings:
+
+* `warning`: The rule specified by the `ruleId` property (&sect;[#issue-ruleId]) was evaluated and a violation was found.
```

Comment:
Not sure if it's still relevant, but I recall that long ago we deliberately avoided the term 'violation' with respect to analysis results. Should we instead say warning -> "an issue was found", error -> "a severe occurrence of an issue was found" or something?